### ISSUE TITLE
How to run ES6 functions?

### ISSUE THREAD

**Issue by rentrop** [SC5/serverless-mocha-plugin#5]
Is there a way to run functions that rely on ES6 and compile them via Bable before testing?

As of now i do get the following error running `sls function mocha-run`.
Do i misinterpret the error and this is unrelated of ES6? 

```
Unhandled rejection SyntaxError: Unexpected reserved word
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/richardrentrop/Desktop/JS/auto-scraper/api/scrape/website/handler.js:4:11)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/richardrentrop/Desktop/JS/auto-scraper/test/scrape-website.js:4:11)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at /Users/richardrentrop/Desktop/JS/auto-scraper/node_modules/mocha/lib/mocha.js:192:27
    at Array.forEach (native)
    at Mocha.loadFiles (/Users/richardrentrop/Desktop/JS/auto-scraper/node_modules/mocha/lib/mocha.js:189:14)
    at Mocha.run (/Users/richardrentrop/Desktop/JS/auto-scraper/node_modules/mocha/lib/mocha.js:422:31)
    at /Users/richardrentrop/Desktop/JS/auto-scraper/node_modules/serverless-mocha-plugin/index.js:122:21
    at tryCatcher (/Users/richardrentrop/Desktop/JS/auto-scraper/node_modules/serverless-mocha-plugin/node_modules/bluebird/js/release/util.js:11:23)
    at Promise._settlePromiseFromHandler (/Users/richardrentrop/Desktop/JS/auto-scraper/node_modules/serverless-mocha-plugin/node_modules/bluebird/js/release/promise.js:489:31)
```

**Comment by mpuittinen** [comment]
@rentrop is this still valid?

**Comment by rentrop** [comment]
Yes it is... I havent found a solution yet.

**Comment by mpuittinen** [comment]
Can you please send me a sample ES6 function and I'll try to reproduce the issue in my environment.

**Comment by mp2526** [comment]
You can reproduce the error by using an import statement instead of a require statement in your handler function. i.e. 

import path from 'path';

**Comment by mp2526** [comment]
If it helps, I got around the issue by calling mocha directly via 

mocha --require babel-register

this requires you to npm install babel-register and make sure you have babel properly configured in your project.

However, I'm not sure how I would go about choosing a stage and region when calling mocha directly. Any thoughts?

**Comment by mpuittinen** [comment]
Solved by #45
